src/frame.js

```diff
@@ -0,0 +1,24 @@
+class Frame {
+  constructor() {
+    this.max_pins = 10
```

Comment:
Is this how to do constants in JS? I genuinely don't know, maybe you can tell me later!

Reply:
I don't know either 😶